cli/src/main.c

```diff
@@ -28,7 +28,8 @@ static const char doc[] =
 	"\v"
 	"Supported commands are:\n"
 	"  discover    Discover Consoles.\n"
-	"  wakeup      Send Wakeup Packet.\n";
+	"  wakeup      Send Wakeup Packet.\n"
+	"  audiostream Fetch the audiostream.\n";
```

Comment:
Imo this should just be called "stream" since you might also want to get video and pipe it somewhere.

Reply:
I've been thinking about this.

If it is possible to fetch the audio-only stream from PS4 (which means that PS4 doesn't even deliver the video stream but only the audio stream), then IMO it makes some sense to have separate commands for `audiostream` and `stream`, because then `audiostream` command won't download the additional video data only to discard it later on. 

If PS4 also force sends us the video stream when we only need the audio stream, then maybe we should just only have the `stream` command like you said which would deliver us both video and audio.